editions/tw5.com/tiddlers/$__editions_tw5.com_related-template.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "created": "20250324175606325",
+        "text": "\\whitespace trim\n<% if [<currentTiddler>has[related]] %>\n\n-----\n<details><summary>Related</summary>\n<ul class=\"tc-related-footer\">\n<$list filter=\"[enlist{!!related}]\"><li><$link/></li></$list>\n</ul>\n</details>\n\n<% endif %>\n\n\n<% if [contains:related<currentTiddler>] [<currentTiddler>backlinks[]] %>\n\n-----\n<details><summary>Linked here</summary>\n<ul class=\"tc-related-footer\"> \n<$list filter=\"[contains:related<currentTiddler>] [<currentTiddler>backlinks[]] +[unique[]sortan[]]\"><li><$link/></li></$list>\n</ul>\n</details>\n\n<% endif %>",
+        "tags": "$:/tags/ViewTemplate",
+        "title": "$:/editions/tw5.com/related-template",
+        "modified": "20250324220838935",
+        "type": "text/vnd.tiddlywiki",
+        "code-body": "yes",
+        "list-after": " $:/core/ui/ViewTemplate/body"
```

Comment:
This is the problematic field, which turned the .tid into .json. There is a leading space at the value. If the space is removed it should be a .tid file again.

Reply:
I will probably close the PR in a day or two and create a new, cleaner version.

The combination of starting this with the PR-maker and then modifying it from the command line, and trying the VS Code git GUI for the first time has made for an... odd experience.  And for a poorly formatted PR, I see.

Before then, if this sounds like a good idea in general, what would we like for the display of this information?  I've done the bare minimum so far, just to prove out the idea.  Is there a useful improvement that shouldn't wait?  And of course, is there other data we should include?